CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing
+
+First of all, thank you for contributing to MeiliSearch! The goal of this document is to provide everything you need to know in order to contribute to MeiliSearch and its different integrations.
+
+<!-- MarkdownTOC autolink="true" style="ordered" indent="   " -->
+
+- [Assumptions](#assumptions)
+- [How to Contribute](#how-to-contribute)
+- [Development Workflow](#development-workflow)
+- [Git Guidelines](#git-guidelines)
+
+<!-- /MarkdownTOC -->
+
+## Assumptions
+
+1. **You're familiar with [GitHub](https://github.com) and the [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)(PR) workflow.**
+2. **You've read the MeiliSearch [documentation](https://docs.meilisearch.com) and the [README](/README.md).**
+3. **You know about the [MeiliSearch community](https://docs.meilisearch.com/resources/contact.html). Please use this for help.**
+
+## How to Contribute
+
+1. Make sure that the contribution you want to make is explained or detailed in a GitHub issue! Find an [existing issue](https://github.com/meilisearch/meilisearch-go/issues/) or [open a new one](https://github.com/meilisearch/meilisearch-go/issues/new).
+2. Once done, [fork the meilisearch-go repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) in your own GitHub account. Ask a maintainer if you want your issue to be checked before making a PR.
+3. [Create a new Git branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository).
+4. Review the [Development Workflow](#workflow) section that describes the steps to maintain the repository.
+5. Make your changes.
+6. [Submit the branch as a PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) pointing to the `master` branch of the main meilisearch-go repository. A maintainer should comment and/or review your Pull Request within a few days. Although depending on the circumstances, it may take longer.<br>
+ We do not enforce a naming convention for the PRs, but **please use something descriptive of your changes**, having in mind that the title of your PR will be automatically added to the next [release changelog](https://github.com/meilisearch/meilisearch-go/releases/).
+
+## Development Workflow
+
+### Install Go
+
+Follow the official [tutorial](https://golang.org/doc/install)
+
+### Install dependencies
+
+```bash
+$ go get -v -t -d ./...
+```
+
+### Tests and Linter
+
+Each PR should pass the tests and the linter to be accepted.
+
+```bash
+# Tests
+$ ./run_tests.sh
+# Install golint if needed (see comment below)
+$ go get -u golang.org/x/lint/golint
+# Use golint
+$ golint
+# Use gofmt
+$ gofmt -w ./..
+```
```

Comment:
### Tests and Linter

Each PR should pass the tests and the linter to be accepted.

#### Tests  <!-- omit in toc -->

Use go test module with your own MeiliSearch instance:

```bash
$ go test -v ./...
```

Or run the tests script which will run a MeiliSearch docker container (uses port 7700):

```bash
$ ./run_tests.sh
```

#### Linter and format  <!-- omit in toc -->

```bash
# Install golint if needed (see comment below)
$ go get -u golang.org/x/lint/golint
# Use golint
$ golint
# Use gofmt
$ gofmt -w ./..
```

Reply:
Last comment was a suggestion, but there is an issue with GitHub interface (cant use ``` inside a suggestion)

Reply:
@eskombro Did you fix the script test? Maybe we should remove it and put the meilisearch command instead, like in the other repos? It's a little bit confusing like that.

Reply:
I agree. I don't know what you mean exactly by "fix the script test". It is working, but no longer used in the CI. I left it there in case someone actually uses it. But just using `go test ./...` should work perfectly now if you have a MeiliSearch running. So yeah, I agree, we can just add the docker command in the readme and ignore the script

Reply:
You perfectly answered my questions. Changing that.